cli/cmd/start/start.go

```diff
@@ -116,15 +126,24 @@ func StartCmd() *cobra.Command {
 			for _, merr := range res.Errors {
 				logger.Errorf("%s: %s", merr.FilePath, merr.Message)
 			}
-			for _, path := range res.AffectedPaths {
-				logger.Infof("Reconciled: %s", path)
+			for _, p := range res.AffectedPaths {
+				logger.Infof("Reconciled: %s", p)
 			}
 			logger.Infof("Hydration completed!")
 
+			installId, err := initGlobalConfig()
+			if err != nil {
+				return err
+			}
+			_, isDev := os.LookupEnv("RILL_IS_DEV")
```

Comment:
Wouldn't it be safer to check the version for this? It's set in `main.go` (defaults to empty, then overridden with linker flags during a release build). You could pass it into the start command and check it.

Reply:
Makes sense.